token-lending/program/src/pyth.rs

```diff
@@ -67,29 +67,34 @@ pub struct PriceComp {
 pub enum PriceType {
     Unknown,
     Price,
-    #[allow(clippy::upper_case_acronyms)]
-    TWAP,
-    Volatility,
 }
 
 #[derive(Copy, Clone)]
 #[repr(C)]
 pub struct Price {
-    pub magic: u32,       // Pyth magic number.
-    pub ver: u32,         // Program version.
-    pub atype: u32,       // Account type.
-    pub size: u32,        // Price account size.
-    pub ptype: PriceType, // Price or calculation type.
-    pub expo: i32,        // Price exponent.
-    pub num: u32,         // Number of component prices.
-    pub unused: u32,
-    pub curr_slot: u64,  // Currently accumulating price slot.
-    pub valid_slot: u64, // Valid slot-time of agg price.
-    pub prod: AccKey,
-    pub next: AccKey,
-    pub agg_pub: AccKey,
-    pub agg: PriceInfo,
-    pub comp: [PriceComp; 16],
+  pub magic      : u32,        // pyth magic number
```

Comment:
Which plugin are you using for this? Imma cop

Reply:
plugin?

Reply:
apparently doing this causes rustfmt to fail anyway LOL